Essentials/src/config.yml

```diff
@@ -656,6 +656,10 @@ command-costs:
 # such as accented letters, you MUST save the file as UTF-8, not ANSI.
 currency-symbol: '$'
 
+# Enable this to make the currency symbol appear behind the currency amount rather than in-front.
+# For example, the euro symbol typically appears after the current amount
```

Comment:
```suggestion
# Enable this to make the currency symbol appear at the end of the amount rather than at the start.
# For example, the euro symbol typically appears after the current amount.
```